plugins/inputs/nvidia_smi/README.md

```diff
@@ -0,0 +1,25 @@
+# `nvidia-smi` Input Plugin
```

Comment:
Can you add the missing sections and update style from: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/EXAMPLE_README.md

Reply:
🙊 I'll add these!

Reply:
This is done!